smartmin/models.py

```diff
@@ -37,15 +36,23 @@ class SmartModel(models.Model):
     created_by = models.ForeignKey(settings.AUTH_USER_MODEL,
                                    related_name="%(app_label)s_%(class)s_creations",
                                    help_text="The user which originally created this item")
-    created_on = models.DateTimeField(auto_now_add=True,
+    created_on = models.DateTimeField(default=timezone.now, editable=False, blank=True,
```

Comment:
`blank=True` and `editable=False` were automatically set by `auto_now` and `auto_now_add` (see https://docs.djangoproject.com/en/1.10/ref/models/fields/#django.db.models.DateField.auto_now)